coda-prototype/ui.html

```diff
@@ -438,5 +435,8 @@ <h1 style="display:none">
 <script src="src/ui/scrollbar.js"></script>
 <script src="src/util/regex.js"></script>
 <script src="src/ui/initUI.js"></script>
+<script src="dist/src/ui/ui.js"></script>
```

Comment:
I hope you've checked that changing the import order is ok here

Reply:
Fixed